app/views/qsets/_qset_list.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="row qset_list">
+  <% @qsets.each_with_index do |qset, index| %>
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h4 class="panel-title">
+          <div class="row valign">
+            <div class="col-sm-11">
+              <span style="cursor:default">
+                <%= link_to(qset) do %>
+                  <%= qset.name %>
+                <% end %>
+                <div class="pull-right">
```

Comment:
This div can be removed, and the `.pull-right` class can be part of the `<span>` on line 13.